providers/redfish/tasks.go

```diff
@@ -126,14 +125,14 @@ func (c *Conn) GetTask(taskID string) (task *gofishrf.Task, err error) {
 
 	resp, err := c.redfishwrapper.Get("/redfish/v1/TaskService/Tasks/" + taskID)
 	if err != nil {
+		if err.Error()[0:3] == "404" {
```

Comment:
```suggestion
		if strings.Contains(err.Error(), "404") {
```

Reply:
I think we should test the beginning of the string so I'll use `strings.HasPrefix`